src/main/resources/db/changelog/changesets/changelog_2022-04-25T10:22:24Z.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-4.1.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet author="garnierlar (generated)" id="1650882159368-8">
+        <createTable tableName="line_split_with_voltage_level">
+            <column name="bbs_or_bus_id" type="VARCHAR(255)"/>
+            <column name="existing_voltage_level_id" type="VARCHAR(255)"/>
+            <column name="line_to_split_id" type="VARCHAR(255)"/>
+            <column name="new_line1id" type="VARCHAR(255)"/>
+            <column name="new_line1name" type="VARCHAR(255)"/>
+            <column name="new_line2id" type="VARCHAR(255)"/>
+            <column name="new_line2name" type="VARCHAR(255)"/>
+            <column name="percent" type="FLOAT8"/>
+            <column name="id" type="UUID">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="line_split_with_voltage_levelPK"/>
+            </column>
+            <column name="may_voltage_level_creation_id" type="UUID"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="garnierlar (generated)" id="1650882159368-9">
+        <addForeignKeyConstraint baseColumnNames="id" baseTableName="line_split_with_voltage_level" constraintName="FKbxsm40mgb8esi1yitlwlxegep" deferrable="false" initiallyDeferred="false" referencedColumnNames="id" referencedTableName="modification" validate="true"/>
+    </changeSet>
+    <changeSet author="garnierlar (generated)" id="1650882159368-10">
+        <addForeignKeyConstraint baseColumnNames="may_voltage_level_creation_id" baseTableName="line_split_with_voltage_level" constraintName="FKgbmxhyxsvkreq7mpimshlj2fl" deferrable="false" initiallyDeferred="false" referencedColumnNames="id" referencedTableName="voltage_level_creation_entity" validate="true"/>
+    </changeSet>
+    <changeSet author="garnierlar (generated)" id="1650882159368-5">
+        <dropNotNullConstraint columnDataType="uuid" columnName="group_id" tableName="modification"/>
+    </changeSet>
+    <changeSet author="garnierlar (generated)" id="1650882159368-6">
+        <dropNotNullConstraint columnDataType="int" columnName="horiz_pos" tableName="voltage_level_creation_entity_busbar_sections"/>
+    </changeSet>
+    <changeSet author="garnierlar (generated)" id="1650882159368-7">
+        <dropNotNullConstraint columnDataType="int" columnName="vert_pos" tableName="voltage_level_creation_entity_busbar_sections"/>
+    </changeSet>
```

Comment:
I don't understand the last 2 changesets (horiz_pos and vert_pos), because no modification has been made on these fields in VoltageLevelCreationEntity ....

Reply:
I had removed some other noisy drop constraints, but missed these two. I remove them and checks if it is still ok, though.